src/actions/actionHelpers.js

```diff
@@ -35,39 +37,39 @@ const sortActionType = (BASE_NAME) => 'SORT_' + BASE_NAME
 const filterActionType = (BASE_NAME) => 'FILTER_' + BASE_NAME
 
 export const paginationActions = (BASE_NAME) => ({
-    types: {
-        GOTO_PAGE: goToPageActionType(BASE_NAME),
-        NEXT_PAGE: nextPageActionType(BASE_NAME),
-        PREV_PAGE: prevPageActionType(BASE_NAME),
-        SORT: sortActionType(BASE_NAME),
-        FILTER: filterActionType(BASE_NAME)
-    },
+  types: {
+    GOTO_PAGE: goToPageActionType(BASE_NAME),
+    NEXT_PAGE: nextPageActionType(BASE_NAME),
+    PREV_PAGE: prevPageActionType(BASE_NAME),
+    SORT: sortActionType(BASE_NAME),
+    FILTER: filterActionType(BASE_NAME)
+  },
 
-    goToPage: (page) => ({
-        type: goToPageActionType(BASE_NAME),
-        page
-    }),
+  goToPage: (page) => ({
+    type: goToPageActionType(BASE_NAME),
+    page
+  }),
 
-    nextPage: () => ({
-        type: nextPageActionType(BASE_NAME)
-    }),
+  nextPage: () => ({
+    type: nextPageActionType(BASE_NAME)
+  }),
 
-    sort: (by) => ({
-        type: sortActionType(BASE_NAME),
-        by
-    }),
+  sort: (by) => ({
+    type: sortActionType(BASE_NAME),
+    by
+  }),
 
-    filter: (filter) => ({
-        type: filterActionType(BASE_NAME),
-        filter
-    })
+  filter: (filter) => ({
+    type: filterActionType(BASE_NAME),
+    filter
+  })
 })
 
-/*Update pagination does nothing on its own, but activates
+/* Update pagination does nothing on its own, but activates
```

Comment:
Is the spacing for comments part of standardJS, or a matter of preference? I don't mind it, just curious.